src/js/education-wizard.js

```diff
@@ -0,0 +1,65 @@
+{
+  function toggleClass(element, className) {
+    element.classList.toggle(className);
+  }
+  function openState(element) {
+    element.setAttribute('data-state', 'open');
+  }
+  function closeState(element) {
+    element.setAttribute('data-state', 'closed');
+  }
+  function isParent(element) {
+    const isParentElement = !element.dataset.selectedForm;
+    return isParentElement;
+  }
+  function closeStateAndCheckChild(alternateQuestion, container) {
+    closeState(alternateQuestion);
+    if (!isParent(alternateQuestion)) {
+      return;
+    }
+    const optionsElements = alternateQuestion.querySelectorAll('input');
+    const options = Array.prototype.slice.call(optionsElements);
```

Comment:
You can use `Array.from()` instead of slice here, since this should now run after we load Babel's ES2105 polyfills.